client/main.js

```diff
@@ -35,6 +35,76 @@ Tabular.getRecord = function (name, collection) {
   return Tabular.getTableRecordsCollection(collection._connection).findOne(name);
 };
 
+Tabular.sanitize = function (data) {
+  const _sanitizeJson = (unsafe) => {
+    if (typeof unsafe !== 'string') {
+      return unsafe;
+    }
+
+    return unsafe.replace(/[&<"']/g, (match) => {
+      switch (match) {
+        case '&':
+          return '&amp;';
+        case '<':
+          return '&lt;';
+        case '>':
+          return '&gt;';
+        default:
+          return match;
+      }
+    });
+  };
+
+  const _sanitizeString = (unsafe) => {
+    return unsafe.replace(/[&<"']/g, (match) => {
+      switch (match) {
+        case '&':
+          return '&amp;';
+        case '<':
+          return '&lt;';
+        case '>':
+          return '&gt;';
+        case '"':
+          return '&quot;';
+        case "'":
+          return '&#039;';
+        default:
+          return match;
+      }
+    });
+  };
+
+  const _isJson = function (value) {
+    value = typeof value !== 'string'
+      ? JSON.stringify(value)
+      : value;
+
+    try {
+      value = JSON.parse(value);
+    } catch (e) {
+      return false;
+    }
+
+    return typeof value === 'object' && value !== null;
+  };
+
+  const sanitizeData = (unsafeArray) => {
+    unsafeArray.forEach(unsafeObject => {
+      for (const key in unsafeObject) {
```

Comment:
@guncebektas is there a specific reason you use `in` instead of `Object.keys` or `Object.entries`? If not, I would please ask you to use one of them instead.